### PR TITLE
Fix Typo in Original English String and All Translations

### PR DESCRIPTION
**Pull request by patrickbaber**
There is a small typo in the word "occured", which misses an additional "r". It has to be "occurred". The string is used as a key for the translations as well, so I replaced all occurrences ;)

### CODE REVIEW

File: settings/js/settings/personalInfo.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ $(document).ready(function () {
 				moment.locale(selectedLocale);
 			},
 			fail: function() {
-				OC.Notification.showTemporary(t('settings', 'An error occured while changing your locale. Please reload the page and try again.'));
+				OC.Notification.showTemporary(t('settings', 'An error occurred while changing your locale. Please reload the page and try again.'));
```
Comment (MorrisJobke) [inline settings/js/settings/personalInfo.js]: Please only update this file. All the translated files will be updated by an automatic script anyways and does not clutter the diff here.

Beside that it looks good 👍

Comment (patrickbaber) [reply settings/js/settings/personalInfo.js]: OK, I changed only the necessary file.

**Review by MorrisJobke — COMMENTED** [review]
Thanks a lot 👍

**Review by violoncelloCH — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**